cross/cyrus-sasl/patches/001.use-host-gcc-to-fix-cross-compile-makemd5.patch

```diff
@@ -0,0 +1,18 @@
+--- include/Makefile.in.org	2018-10-09 14:58:13.000000000 +0000
```

Comment:
I would reserve "dot" character for file "extension", so please rename file as "001_use-..."